bin/powershell/metric-windows-cpu-load.ps1

```diff
@@ -23,45 +23,42 @@
 #   Released under the same terms as Sensu (the MIT license); see LICENSE for details.
 #
 
+param(
+    [switch]$UseFullyQualifiedHostname
+    )
+
 $ThisProcess = Get-Process -Id $pid
 $ThisProcess.PriorityClass = "BelowNormal"
 
 . (Join-Path $PSScriptRoot perfhelper.ps1)
 
-$ThisProcess = Get-Process -Id $pid
-$ThisProcess.PriorityClass = "BelowNormal"
+if ($UseFullyQualifiedHostname -eq $false) {
+    $Path = ($env:computername).ToLower()
+}else{
```

Comment:
apologies if that is how people in powershell as I know almost nothing about it. My developer senses are tingling and telling me that there either should be a space between `}else` if not a newline.